compute/gpu/how-to/use-gpu-with-docker.mdx

```diff
@@ -0,0 +1,203 @@
+---
+meta:
+  title: How to access the GPU using Docker
+  description: This page explains how to access the GPU using Docker
+content:
+  h1: How to access the GPU using Docker
+  paragraph: This page explains how to access the GPU using Docker
+dates:
+  validation: 2022-03-25
+  posted: 2022-03-25
+---
+
+Docker is a platform as a service (PaaS) tool that uses OS-level virtualization to deliver applications in packages called containers. Each container is isolated from the others and each of them bundles their own software, libraries, and configuration files. 
+
+Unlike virtual machines, containers share the services of a single operating system kernel. This reduces unnecessary overhead and makes them lightweight and portable. Docker containers can run on any computer running macOS, Windows, or Linux, either on-premises or in a public cloud environment, such as Scaleway Elements.
+
+All [Scaleway GPU Instances](https://www.scaleway.com/en/gpu-instances/) come with prebuilt Docker images which can be launched as soon as you connect to your Instance. Each image provides a different AI environment. When you launch one of these images, you are in your chosen environment within seconds with all your favourite python packages already installed. Using Docker for your AI projects in this way allows you to ensure that your working environments are both **isolated** and **portable**, since they are in containers which can be easily transferred between machines.
```

Comment:
```suggestion
All [Scaleway GPU Instances](https://www.scaleway.com/en/gpu-instances/) come with prebuilt Docker images which can be launched as soon as you connect to your Instance. Each image provides a different AI environment. When you launch one of these images, you are in your chosen environment within seconds with all your favorite Python packages already installed. Using Docker for your AI projects in this way allows you to ensure that your working environments are both **isolated** and **portable**, since they are in containers which can be easily transferred between machines.
```